src/controllers/handlers/livekit-webhook-handler.ts

```diff
@@ -41,31 +45,45 @@ export async function livekitWebhookHandler(
     }
   } else if (event === 'ingress_ended' && webhookEvent.ingressInfo) {
     await sceneStreamAccessManager.stopStreaming(webhookEvent.ingressInfo.ingressId)
-  } else if (event === 'participant_joined' && isVoiceChatRoom) {
-    if (!webhookEvent.participant?.identity) {
-      throw new InvalidRequestError("LiveKit didn't provide an identity for the participant")
-    }
-    if (!webhookEvent.room?.name) {
-      throw new InvalidRequestError("LiveKit didn't provide a room name for the participant")
-    }
+  } else if (event === 'participant_joined') {
+    analytics.fireEvent(AnalyticsEvent.PARTICIPANT_JOINED_ROOM, {
+      room: webhookEvent.room?.name ?? 'Unknown',
+      address: webhookEvent.participant?.identity ?? 'Unknown'
+    })
 
-    logger.debug(`Participant ${webhookEvent.participant.identity} joined voice chat room ${webhookEvent.room.name}`)
+    if (isVoiceChatRoom) {
+      if (!webhookEvent.participant?.identity) {
+        throw new InvalidRequestError("LiveKit didn't provide an identity for the participant")
+      }
+      if (!webhookEvent.room?.name) {
+        throw new InvalidRequestError("LiveKit didn't provide a room name for the participant")
+      }
```

Comment:
WDYT about moving these validations to a new abstraction so we can reuse them on different events?